builder/scaleway/step_create_server.go

```diff
@@ -60,6 +62,27 @@ func (s *stepCreateServer) Run(ctx context.Context, state multistep.StateBag) mu
 		return multistep.ActionHalt
 	}
 
+	ui.Say("Waiting for any user data apply to finish if provided...")
+
+	m := map[string]io.Reader{}
+	for k, v := range c.UserData {
+		m[k] = bytes.NewBufferString(v)
+	}
+
+	createUserDataReq := &instance.SetAllServerUserDataRequest{
+		Zone:     scw.Zone(c.Zone),
+		ServerID: createServerResp.Server.ID,
+		UserData: m,
+	}
+
+	err = instanceAPI.SetAllServerUserData(createUserDataReq, scw.WithContext(ctx))
```

Comment:
Should the server be rebooted to have all of this applied? I'm wondering when the code is taken into account